src/nupic/regions/PyRegion.py

```diff
@@ -177,6 +177,15 @@ def getOutputElementCount(self, name):
     name: the name of the output
     """
 
+  def getAlgorithm(self):
```

Comment:
Should we rename it to be a bit more precise, such as getAlgorithmInstance or getAlgorithmObject ?